tests/testConvolve.py

```diff
@@ -39,7 +39,7 @@
 import numpy
 
 import lsst.utils
-import lsst.utils.tests as utilsTests
+import lsst.utils.tests
 import lsst.pex.logging as pexLog
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
```

Comment:
Oh, fun. Tests that directly import other tests. Yuck. I'd suggest moving all of those functions that are wanted by multiple tests here into a kernelTestHelpers.py or something in this directory (with a docstring saying what it's for), and then importing that.